src/IB/ConstraintIBMethod.cpp

```diff
@@ -275,52 +275,42 @@ ConstraintIBMethod::ConstraintIBMethod(std::string object_name,
 
         for (int struct_no = 0; struct_no < d_no_structures; ++struct_no)
         {
-            std::ostringstream trans_vel, rot_vel, drag_force, moment_inertia, torque, position_com, power_spent;
-
-            trans_vel << d_base_output_filename + "_Trans_vel" + "_struct_no_" << struct_no;
-            rot_vel << d_base_output_filename + "_Rot_vel" + "_struct_no_" << struct_no;
-            drag_force << d_base_output_filename + "_Drag_force" + "_struct_no_" << struct_no;
-            moment_inertia << d_base_output_filename + "_MOI" + "_struct_no_" << struct_no;
-            torque << d_base_output_filename + "_Torque" + "_struct_no_" << struct_no;
-            position_com << d_base_output_filename + "_COM_coordinates" + "_struct_no_" << struct_no;
-            power_spent << d_base_output_filename + "_Power_spent" + "_struct_no_" << struct_no;
-
             if (from_restart)
-                d_trans_vel_stream[struct_no] = new std::ofstream(trans_vel.str().c_str(), std::fstream::app);
+                d_trans_vel_stream[struct_no] = new std::ofstream(d_base_output_filename + "_Trans_vel" + "_struct_no_" + std::to_string(struct_no), std::fstream::app);
             else
```

Comment:
We can combine the two string literals here and below, i.e., write `d_base_output_filename + "_Trans_vel_struct_no_"`